app/models/geocoding.rb

```
@@ -123,6 +123,7 @@ def run!
 
     self.run_geocoding!(processable_rows, rows_geocoded_before)
   ensure
+    table_geocoder.reset_connection
```

I realize there shouldn't be a geocoder without a user so I'd prefer to remove the `user.present?` check for both cases and fail fast.